tests/wamr-test-suites/test_wamr.sh

```diff
@@ -362,31 +362,31 @@ function sightglass_test()
 function setup_wabt()
 {
     # please sync with .github/actions/install-wasi-sdk-wabt/action.yml
+    case ${PLATFORM} in
+        cosmopolitan)
+            ;;
+        linux)
+            WABT_URL=https://github.com/WebAssembly/wabt/releases/download/1.0.37/wabt-1.0.37-ubuntu-20.04.tar.gz
+            WABT_VERSION=1.0.37
+            ;;
+        darwin)
+            WABT_URL=https://github.com/WebAssembly/wabt/releases/download/1.0.36/wabt-1.0.36-macos-12.tar.gz
+            WABT_VERSION=1.0.36
+            ;;
+        windows)
+            WABT_URL=https://github.com/WebAssembly/wabt/releases/download/1.0.37/wabt-1.0.37-windows.tar.gz
+            WABT_VERSION=1.0.37
+            ;;
+        *)
+            echo "wabt platform for ${PLATFORM} in unknown"
+            exit 1
```

Comment:
why have you moved the check out of the WABT_BINARY_RELEASE block below?

Reply:
WABT_VERSION is also used in else branch(L398 ~ L411)

Reply:
so, it wasn't working before this change?

Reply:
ok